Bangazon/Bangazon/urls.py

```diff
@@ -6,6 +6,7 @@
 # Create a router, using DefaultRouter
 # Register each ViewSet with it.
 router = DefaultRouter()
+router.register(r'staff', views.UserStaffViewSet)
```

Comment:
This naming convention might be confusing, as it seems to be listing staff members. How about `auth` and `unauth` or something?